Documentation/project-docs/target-dotnetcore-with-msbuild.md

```diff
@@ -0,0 +1,119 @@
+Using MSBuild to build .NET Core projects
+=========================================
+
+The .NET Core tooling is going to [move from project.json to MSBuild based projects](https://blogs.msdn.microsoft.com/dotnet/2016/05/23/changes-to-project-json/).
+We expect the first version of the .NET Core tools that use MSBuild to ship along with Visual Studio "15".  However, it is possible to use MSBuild for .NET Core
+projects today, and this page shows how.
+
+We recommend that most people targeting .NET Core today use the default tooling experience with project.json.  This is because we haven't yet added support to MSBuild
```

Comment:
The keyword for me is "new". I don't want to recommend existing customers with msbuild projects switch to project.json.